pending/tg.js

```diff
@@ -0,0 +1,777 @@
+windows.host.rewards Pinipilit ng pagpapagana ng KR Rewards ang Server na gumamit ng Default na Mga Setting
+windows.store.wheels.sub Subukan ang iyong kapalaran at paikutin para sa isang pagkakataon upang manalo ng ilang mga bagong disenyo!
+windows.store.currency * Ang mga presyo sa USD ay maaaring mai-convert sa iyong lokal na pera sa isang rate ng palitan na tinukoy ng provider ng pagbabayad o bangko
+windows.store.creator Isaalang-alang ang Suportahan ang iyong paboritong Tagalikha ng Nilalaman sa pamamagitan ng paggamit ng isang Creator Code
+windows.chall.customs Ang mga hamon ay hindi nakukumpleto sa mga kustom na laro
+twitch.link Sigurado ka bang nais mong i-link ang twitch account?
+twitch.unlink Sigurado ka bang nais mong i-unlink ang iyong twitch account?
+windows.clan.leave Sigurado ka bang nais mong umalis sa hukbo/samahan ninyo?
+windows.clan.delete Sigurado ka bang nais mong tanggalin ang hukbo/samahan?
+windows.clan.kick Sigurado ka bang nais mong i-kick si {0} at tanggalin sa samahan?
+windows.store.premium.purchase Sigurado ka bang nais mong bumili ng {0} araw ng Premium?
+windows.playerlist.ban Sigurado ka bang nais mong ipagbawal si {0} mula sa larong ito?
+windows.playerlist.flag Sigurado ka bang nais mong I-flag {0}?
+windows.playerlist.unflag Sigurado ka bang nais mong mag-Unflag {0}?
+windows.playerlist.punish Sigurado ka bang nais mong parusahin ang manlalaro {0}?
+item.view.open Buksan ang Model Viewer
+item.view.more Iba pang Impormasyon
+item.sell.cant Hindi mabenta ang kagamitan.
+item.sell.quick Sigurado ka bang ibebenta mo ito agad {0}?
+social.profile.stats Kabuoang statistika
+social.profile.class Statistika ng mga Klase
+social.profile.sales Mga Binebenta
+social.profile.sales.market Mga Binebenta sa Market
+social.profile.skins Mga Skins o Disenyo [{0}]
+social.profile.skins.none Walang inakdang mga Skins o Disenyo mula sa User
+social.profile.games Mga Laro [{0}]
+social.profile.games.none Walang inakdang Laro o Mapa mula sa User
+social.profile.assets Mga Assets [{0}]
+social.profile.assets.none Walang inakdang Assets mula sa User
+social.profile.mods Mga Mods [{0}]
+social.profile.mods.none Walang inakdang Mods mula sa User
+social.search Resulta ng Paghahanap
+social.market.stats.rare Bihirang Mahanap na Items
+social.market.stats.circulation Nasa Sirkulasyon
+social.market.stats.units Bilang ng Item sa Daigdig
+social.trades.in Pumapasok
+social.trades.in.head Pumapasok na Pag-aalok ({0})
+social.trades.out Palabas
+social.trades.out.head Palabas na Pag-aalok ({0})
+social.trades.history Kasaysayan
+social.trades.history.head Kasaysayan ng Pag-kakalakalan ({0})
+social.trades.decline Tanggihan Lahat
+social.trades.message Payagan ang mga Trade Message at Pag-alok sa Settings
+social.trades.warning Wala kang makukuhang kagamitan sa Pag-alok na ito!
+social.trades.accept Sigurado ka bang nais mong tanggapin ang Pag-alok na ito?
```

Comment:
```suggestion
social.trades.accept Nais mo bang tanggapin ang Pag-alok na ito? or "Sigurado ka bang tatanggapin ang Pag-alok na ito?"

```

Use only one of the 2 to shorten the length of the sentence.